ush/global_ens/evs_get_gens_atmos_data.sh

```diff
@@ -447,7 +447,12 @@ if [ $modnam = ccpa ] ; then
        ${METPLUS_PATH}/ush/run_metplus.py -c ${PARMevs}/metplus_config/machine.conf -c ${CONF_PREP}/PcpCombine_obsCCPA24h.conf
        cp $output_base/ccpa.t12z.grid3.24h.f00.nc $COMOUT_gefs/.
     else
-       echo "At least one of ccpa06h files is missing!"
+       export subject="06h CCPA Data Missing for 24h CCPA generation"
+       export maillist=${maillist:-'geoffrey.manikin@noaa.gov,binbin.zhou@noaa.gov'}
```

Comment:
@BinbinZhou-NOAA this line (451) needs to be removed now that the variable is exported from the ecf script.